themes/default/gulpfile.js

```diff
@@ -17,6 +17,9 @@ var gulp = require('gulp'),
 	concat = require('gulp-concat'),
 	uglify = require('gulp-uglify'),
 	plumber = require('gulp-plumber');
+	rework = require('gulp-rework'),
```

Comment:
In my experience this is the first thing we change for _every_ project. Is the overhead of this being done by default very detrimental to compile times?